### PR TITLE
Aks cheatsheet

### PR DESCRIPTION
**Pull request by tomas-stefano**
## Context

After the AKS meeting where I shown all the commands, I am opening this PR which is a document of the AKS cheatsheet to help everyone.

In the future we will have all make commands that will be wrapper to this but while we don't have this can be useful.

### CODE REVIEW

File: docs/aks-cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+All examples below show qa usage and you should adapt accordingly.
+
+## Authentication
+
+### Raising a PIM request
+
+You need to activate the role in the desired cluster below:
+https://portal.azure.com/?Microsoft_Azure_PIMCommon=true#view/Microsoft_Azure_PIMCommon/ActivationMenuBlade/~/azurerbac
+
+Example: Activate `*-teacher-services-cloud-test`. It will be approved automatically after a few seconds
```
Comment (stevehook) [inline docs/aks-cheatsheet.md:14]: Is there a way to do this from the command-line? (I remember finding the Azure Web interface very confusing!)

File: docs/aks-cheatsheet.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+Example of loading test:
+
+```
+bin/konduit.sh apply-loadtest -- psql
+```
+
+## More info
+
+For more info see
+[Kubernetes cheatsheet](https://kubernetes.io/docs/reference/kubectl/cheatsheet/)
```
Comment (stevehook) [inline docs/aks-cheatsheet.md:152]: :+1: was just about to ask whether there are any external references about k8s?

**Review by stevehook — APPROVED** [review]
This is already better than the PasS cheatsheet because we never added instructions for conduit there!

File: docs/aks-cheatsheet.md (hunk 1 above)
Comment (tomas-stefano) [reply docs/aks-cheatsheet.md:14]: No, only if we make a command that called open that url 😅

**Pull request merged**